OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -461,20 +461,24 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
             DreamProcArguments arguments = new DreamProcArguments(new(), new());
 
             if (state.Pop().TryGetValueAsDreamList(out var argList)) {
```

Comment:
This is no longer used.
```suggestion
            if (state.Pop().TryGetValueAsDreamList(out var argList)) {
```